<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_bearb_bohrung</class>
 <widget class="QDialog" name="Dialog_bearb_bohrung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>540</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_afb"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Pos X</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Pos Z</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Ausführbedingung</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Durchmesser</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Zustellmaß</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_wkz"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_ti"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Werkzeugnummer</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_dm"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_y"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Tiefe</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="comboBox_bezug"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_zust"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_z"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Pos Y</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_x"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Bezug</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_invert">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>HBE invertieren</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_dm</tabstop>
  <tabstop>lineEdit_ti</tabstop>
  <tabstop>lineEdit_x</tabstop>
  <tabstop>lineEdit_y</tabstop>
  <tabstop>lineEdit_z</tabstop>
  <tabstop>lineEdit_zust</tabstop>
  <tabstop>comboBox_bezug</tabstop>
  <tabstop>lineEdit_afb</tabstop>
  <tabstop>lineEdit_wkz</tabstop>
  <tabstop>btn_ok</tabstop>
  <tabstop>btn_abbrechen</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
